### ISSUE TITLE
Suggestion: Deprecate d3graph and have it be part of d3blocks

### ISSUE THREAD

**Issue by JohnOmernik** [erdogant/d3graph#20]
I think while it may cause a bit of pain at first, the updates/maintenance of  both projects would gain value from being moved together.

**Comment by erdogant** [comment]
I agree this will be the best move in the long run. I just need to find the time to do it ;)
